config/application.rb

```diff
@@ -62,9 +59,9 @@ class Application < Rails::Application
     end
 
     config.middleware.insert_before(0, HttpMethodNotAllowed)
-    config.middleware.use 'OliveBranch::Middleware'
-    config.middleware.use 'StatsdMiddleware'
-    config.middleware.use 'Rack::Attack'
+    config.middleware.use OliveBranch::Middleware
```

Comment:
It think changing these from strings to classes will work in 4.2 as well could this go in a PR that gets merged in before this one? (with any other changes that are both 4.2 and 5.0 compatible)

Reply:
I've opened a PR [here](https://github.com/department-of-veterans-affairs/vets-api/pull/2893) to do so. CC: @lihanli